src/core/include/antioch/chemical_mixture.h

```diff
@@ -569,6 +585,24 @@ namespace Antioch
     return;
   }
 
+  template<typename CoeffType>
+  inline
+  const ChemicalMixture<CoeffType> & ChemicalMixture<CoeffType>::chemical_mixture() const
+  {
+     antioch_deprecated();
+     return *this;
+  }
+
+  template<typename CoeffType>
+  template<typename ThermoEvaluator>
+  inline
+  const ThermoEvaluator & ChemicalMixture<CoeffType>::thermo_evaluator() const
+  {
+     std::cerr << "What the hell are you doing here? This backward compatibility place should not be explored" << std::endl;
```

Comment:
Hmm... github didn't show me the "antioch_error();" below it.  That fixes the spamminess problem, but just raises further questions:

This is a backwards compatibility shim that's been created to fail at run time rather than compile time?  That's not an improvement...